playwright/tests/itemDetailPageElements.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { Given, Then } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I am on an item detail page", async function (this: CustomWorld) {
+  return await this.page.goto(
+    `${this.parameters.appUrl}/work/01a28167-8c8d-4141-a32f-718539d5c8a4?featured=949699`
+  );
+});
+
+Then("I see the item title", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("#work-title", { hasText: "Introduction to Africa" })
+  ).toBeVisible();
+});
+
+Then("I see the item author", async function (this: CustomWorld) {
+  await expect(this.page.locator("text=By Library of Congress")).toBeVisible();
+});
+
+Then("I see the featured edition heading", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("#featured-edition", {
+      hasText: "Featured Edition",
+    })
+  ).toBeVisible();
+});
+
+Then("I see multiple cover images", async function (this: CustomWorld) {
+  const coverimage = this.page.locator('[data-testid="test"]');
+  expect(await coverimage.count()).toBeGreaterThan(3);
+});
+
+Then("I see the edition year", async function (this: CustomWorld) {
+  await expect(this.page.locator("//h4 >> nth=0")).toContainText("Edition");
+});
+
+Then("I see the edition publisher", async function (this: CustomWorld) {
+  const publishedby = this.page.locator(
+    '//div[contains(string(), "Published by")]'
+  );
+  expect(await publishedby.count()).toBeGreaterThan(10);
+});
+
+Then("I see the edition language", async function (this: CustomWorld) {
+  const language = this.page.locator('//div[contains(string(), "Languages")]');
+  expect(await language.count()).toBeGreaterThan(10);
+});
+Then("I see the edition license", async function (this: CustomWorld) {
+  const license = this.page.locator('//div/a[contains(string(), "License")]');
+  expect(await license.count()).toBeGreaterThan(2);
+});
+
+Then("I see the details heading", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("#details-list-heading", {
+      hasText: "Details",
+    })
+  ).toBeVisible();
+});
+
+Then("I see the authors heading", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//dt[contains(text(),'Authors')]")
+  ).toBeVisible();
+});
+Then("I see the author", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator(
+      "//dt[contains(text(),'Authors')]/following-sibling::dd/a[contains(text(),'Library of Congress')]"
```

Comment:
This xpath seems like it could be brittle but I understand that there might not be a better solution.